modules/services/dropbox.nix

```
@@ -0,0 +1,73 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.dropbox;
+  baseDir = ".dropbox-hm";
+in {
+  meta.maintainers = [ maintainers.eyjhb ];
+
+  options = {
+    services.dropbox = {
+      enable = mkEnableOption "Dropbox daemon";
+
+      path = mkOption {
+        type = types.path;
+        default = "${config.home.homeDirectory}/Dropbox";
+        defaultText = "\${config.home.homeDirectory}/Dropbox";
+        description = "Where to put the Dropbox directory.";
+      };
+    };
+  };
+
+  config = mkIf cfg.enable (mkMerge [{
+    home.packages = [ pkgs.dropbox-cli ];
+
+    systemd.user.services.dropbox = {
+      Unit = { Description = "Starting dropbox"; };
+
+      Install = { WantedBy = [ "default.target" ]; };
+
+      Service = {
+        Environment = [ "HOME=${config.home.homeDirectory}/${baseDir}" ];
+
+        Type = "forking";
+        PIDFile =
+          "${config.home.homeDirectory}/${baseDir}/.dropbox/dropbox.pid";
+
+        Restart = "on-failure";
+        PrivateTmp = true;
+        ProtectSystem = "full";
+        Nice = 10;
+
+        ExecReload = "${pkgs.coreutils.out}/bin/kill -HUP $MAINPID";
+        ExecStop = ''
+          ${pkgs.coreutils}/bin/env -i HOME="$HOME" ${pkgs.dropbox-cli}/bin/dropbox stop'';
+        ExecStart = let
+          script = pkgs.writeScript "dropboxInit" ''
+            #!/bin/sh
+                        if [ ! -f $HOME/.dropbox-dist/VERSION ]; then
+                          ${pkgs.coreutils}/bin/printf "y\n" | ${pkgs.coreutils}/bin/env -i HOME="$HOME" ${pkgs.dropbox-cli}/bin/dropbox update
+                        fi
+
+                        ${pkgs.coreutils}/bin/env -i HOME="$HOME" ${pkgs.dropbox-cli}/bin/dropbox start
+                    '';
+        in "${script}";
+      };
+    };
+
+    home.activation.dropbox = let dag = lib.hm.dag;
+    in (dag.entryAfter [ "writeBoundary" ] ''
```

I don't think you need any parentheses here
```suggestion
    in dag.entryAfter [ "writeBoundary" ] ''
```